tests/playwright/resources/squid-proxy/generate-certs.sh

```
@@ -0,0 +1,90 @@
+# After running the script, the CA will be in ${destDir}/rootCA.crt, clients
+# will need it, but the squid container has no need for it
+# The only file squid needs is ${destDir}/squid.pem
+#
+# The certificate is only valid for 127.0.0.1, localhost and 192.168.122.1
+# If certificates for other hostnames/IPs are needed, modify the [alt_names]
+# section
+#
+# The certificate used by squid can be overriden when starting the container:
+# podman run -v ./pki/:/etc/pki/squid:z ...
+
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)